features/step_definitions/cucumber_cpp_mappings.rb

```diff
@@ -172,36 +189,34 @@ def assert_executed_scenarios *scenario_offsets
   end
 
   def assert_world_variable_held_value_at_time(value, time)
-    check_exact_file_content "#{WORLD_VARIABLE_LOG_FILE}.#{time}", value
+    expect(File.read("#{WORLD_VARIABLE_LOG_FILE}.#{time}")).to match value 
   end
 
   def assert_world_function_called
-    check_file_presence [WORLD_FUNCTION_LOG_FILE], true
+    expect(File.file?(WORLD_FUNCTION_LOG_FILE)).to be true
   end
 
   def assert_cycle_sequence *args
     expected_string = args.join CYCLE_SEQUENCE_SEPARATOR
-    check_file_content(CYCLE_LOG_FILE, expected_string, true)
+    expect(File.read(CYCLE_LOG_FILE)).to match expected_string
   end
 
   def assert_cycle_sequence_excluding *args
     args.each do |unexpected_string|
-      check_file_content(CYCLE_LOG_FILE, unexpected_string, false)
+      expect(File.read(CYCLE_LOG_FILE)).not_to match unexpected_string
     end
   end
 
   def assert_complete_cycle_sequence *args
     expected_string = "#{CYCLE_SEQUENCE_SEPARATOR}#{args.join(CYCLE_SEQUENCE_SEPARATOR)}"
-    check_exact_file_content(CYCLE_LOG_FILE, expected_string)
+    expect(File.read(CYCLE_LOG_FILE)).to match expected_string
   end
 
   def assert_data_table_equals_json(json)
-    prep_for_fs_check do
-      log_file_contents = IO.read(DATA_TABLE_LOG_FILE)
-      actual_array      = JSON.parse(log_file_contents)
-      expected_array    = JSON.parse(json)
-      actual_array.should == expected_array
-    end
+    log_file_contents = File.read(DATA_TABLE_LOG_FILE)
+    actual_array      = JSON.parse(log_file_contents)
+    expected_array    = JSON.parse(json)
+    expect(actual_array).to be == expected_array
```

Comment:
`prep_for_fs_check`  is deprecated: http://www.rubydoc.info/github/cucumber/aruba/Aruba%2FApi%2FDeprecated%3Aprep_for_fs_check